platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java

```diff
@@ -212,7 +212,7 @@ private Bundle getBoundBundle(Configuration config) {
                     + service.getId()
                     + "_disabled))");
         if (configs != null) {
-          addConfigurationData(service, configs);
+          addConfigurationData(service, configs, getPidObjectClasses(metaTypeInformationByBundle));
```

Comment:
✏️  I know you got this patch from me but something I forgot to do was to make a local variable to hold the results from `getPidObjectClasses`.  It is now called and used 3 times in this method.

Reply:
Updated 👍